lib/routes/google/research.ts

```diff
@@ -0,0 +1,45 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const baseUrl = `https://research.google`;
+
+export const route: Route = {
+    path: '/research',
+    categories: ['blog'],
+    example: '/research',
+    name: 'Google Research Blog',
+    maintainers: ['Levix'],
+    radar: [
+        {
+            source: [baseUrl],
```

Comment:
The value of source should be: `['research.google']`

Reply:
fix